Hometask_4_2/Hometask_4_2.py

```diff
@@ -0,0 +1,51 @@
+import random
+import string
+
+
+# Create the list dictionaries from 2 to 10.
+def create_dicts():
+    output = []  # Dicts list
+    for _ in range(random.randint(2, 10)):  # Dictionary counter
+        dictionary = {}
+        for _ in range(random.randint(1, 26)):  # Elements counter
+            # Key(random letter), value(random number from 0 to 100)
+            dictionary[random.choice(string.ascii_lowercase)] = random.randint(0, 100)
+        output.append(dictionary)  # Append every dictionary to the list
+    return output
+
+
+all_values = create_dicts()
+
+
+def all_values_dict():
+    output = {}
+    c = 1
+    for cntr in all_values:
+        for key, value in cntr.items():
+            if key not in output:  # Check whether value exists previously
+                output[key] = {str(c): value}
+            else:
+                output[key][str(c)] = value
+        c += 1
+    return output
+
+
+all_values_dict = all_values_dict()
+
+
+def final_dict():
+    my_dict = {}
+    for key, value in all_values_dict.items():
+        if len(all_values_dict[key]) == 1:
+            my_dict[key] = max(all_values_dict[key].values())
+        else:
+            my_dict[key + "_" + str(max(all_values_dict[key], key=all_values_dict[key].get))] = max(all_values_dict[
```

Comment:
This row is to long. 
You can use separate variable for that:
`key_with_max_value = key + "_" + str(max(all_values_dict[key], key=all_values_dict[key].get))`